main_server/local_server/server_admin/c_comdef_admin_main_console.class.php

```diff
@@ -381,7 +381,9 @@ public function return_main_console_html()
                 $ret .= 'var g_naws_popup_prompt = \''.self::js_html($this->my_localized_strings['comdef_server_admin_strings']['world_format_codes_prompt']).'\';'.(defined('__DEBUG_MODE__') ? "\n" : '');
                 $ret .= "var g_naws_values = [";
                     $n_first = true;
-            ksort($this->my_localized_strings['comdef_server_admin_strings']['world_format_codes']);
```

Comment:
This looks good, but it'd be helpful to put a comment in the code explaining why you're doing what you're doing

Reply:
ok I added notes.